common-libs/common/src/services/RegistryResolveDidService.ts

```diff
@@ -0,0 +1,33 @@
+const RESOLVE_DID_PATH = '/api/v1/did/resolve-did'
+
+let fetch: any
+
+if (!fetch) {
+  fetch = require('node-fetch')
+}
+
+export class RegistryResolveDidService {
```

Comment:
Although it doesn't seem to be used anywhere?

Reply:
yes, removed=)